prod/README.md

```diff
@@ -4,14 +4,17 @@ This directory contains everything required to deploy MIQA in production using d
 # Instructions
 All of these commands should be run from the `prod` directory.
 
-### Set the variables in .env
-The `.env` file contains some default values for local testing that will need to overwritten for production deployments.
-For the build and setup process these values are mostly placeholders, but they will need to be set correctly before the site is made public.
+### Setup the configuration variables
 
-`DJANGO_EMAIL_URL` must be of the form `submission://username:password@my.smtp.server:port`. See the [documentation](https://github.com/migonzalvar/dj-email-url#supported-backends) for details with encoding special characters.
+Run:
 
-You can also specify those variables explicitly in your local shell environment instead of editing the file.
-`docker-compose` will check for local shell values before defaulting to the values in `.env`.
+`cp .env.template .env`
+
+Open the `.env` file and make changes to the variables in the top section.
+
+* `DJANGO_EMAIL_URL` must be of the form `submission://username:password@my.smtp.server:port`. See the [documentation](https://github.com/migonzalvar/dj-email-url#supported-backends) for details with encoding special characters.
+
+You can also specify those variables explicitly in your local shell environment instead of editing the file. `docker-compose` will check for local shell values before defaulting to the values in `.env`.
```

Comment:
Just checking, but  `docker-compose` won't override variables in the local shell with values in `.env` correct? Will it only default to values in `.env` if they are undefined/empty?

Reply:
I actually haven't tested this. I thought about deleting it but wasn't sure. Maybe I'll just delete it.

Reply:
> Values in the shell take precedence over those specified in the .env file.

\- https://docs.docker.com/compose/environment-variables/